cg/meta/workflow/nextflow_common.py

```diff
@@ -35,6 +35,13 @@ def get_case_config_path(cls, case_id: str, root_dir: str) -> Path:
         """Generates a path where the Rnafusion sample sheet for the case_id should be located."""
         return Path((cls.get_case_path(case_id, root_dir)), f"{case_id}_samplesheet.csv")
 
+    @classmethod
+    def get_case_pid_path(cls, case_id: str, root_dir: str) -> Path:
+        """Generates a path where the Nextflow pid file for the case_id should be located."""
+        # If not specified with the NXF_PID_FILE variable, a .nextflow.pid is created in the launch directory when
+        # running nextflow in the background (with the bg option)
+        return Path((cls.get_case_path(case_id, root_dir)), f"{case_id}_nextflow.pid")
```

Comment:
Use named arguments. It improves readability and help convey what the function expects from the caller